protoc-gen-openapiv2/defs.bzl

```diff
@@ -119,6 +120,9 @@ def _run_proto_gen_openapi(
     if proto3_optional_nullable:
         args.add("--openapiv2_opt", "proto3_optional_nullable=true")
 
+    if visibility_restriction_selectors:
+        args.add("--openapiv2_opt", "visibility_restriction_selectors=%s" % visibility_restriction_selectors)
```

Comment:
I assume we actually need to loop over `visibility_restriction_selectors` now and add a new argument for each element, instead of this string interpolation?